installscripts/wizard/scenarios/stack_with_bb_jenkins.py

```diff
@@ -19,44 +19,44 @@ def check_jenkins_pem():
         " Please make sure that you have the ssh login user names of jenkins and bitbucket servers."
     )
     print(
-        " Please create jenkinskey.pem with private keys of Jenkins Server in your home directory"
+        " Please place the private key to your Jenkins server in the installer root directory with a filenameof 'jenkinskey.pem"
     )
 
     pause()
 
     # Check if file is been added to home derectory
-    jenkins_pem = HOME_FOLDER + "/jenkinskey.pem"
+    jenkins_pem =  get_installer_root() + "/jenkinskey.pem"
```

Comment:
@bleggett Got one error, 

> "File jenkinskey.pem not found in installer root directory, kindly add and run the installer again!"

Actually the pem key is present in the "/home/centos" path. So the path here should be HOME_FOLDER.
@suryajak Please confirm.

Reply:
The thing I was trying to avoid here was hardcoding assumptions about where this should be run from.

If I want to run the installer from `/tmp/jazz_test`, should I be required to put the key in the home folder of the user I am running the script as, or should I just drop it in the repo after cloning it?

----
One other thing that I may not have made clear: You do not need to clone the repo before running `centos7-provision.sh`, you can `curl` that script and run it to clone the repo to a `jazz_installer` subfolder, just like the old `Installer.sh`.

If you have manually cloned the repo, and the prerequisites are already installed, you can skip that and just run `Installer.sh` from the cloned repo root.

I tried to keep the `curl`-ability of the old script intact, but I think the cleanest approach would be to remove the `git clone` step from the `centos7-provision.sh` entirely, and have that be a manual step, so: 1. Clone repo 2. (optional) Run centos7 provisioning script from repo root 3. Run installer script from repo root.

Again, I did not do that in this PR because I figured that would require too many changes to your test setup.

Reply:
@bleggett Yes, we should keep the jenkins pem key in the in the repo root folder after cloning it. Otherwise it will throw error

Reply:
@rajeevr2715 : Is this error in v1.4.6 scenario 1?

Reply:
Yes